src/parser/mod.rs

```diff
@@ -7070,13 +7071,22 @@ impl<'a> Parser<'a> {
         }
     }
 
-    /// Parse configuration like partitioning, clustering information during the table creation.
+    /// Parse configuration like inheritance, partitioning, clustering information during the table creation.
     ///
     /// [BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#syntax_2)
-    /// [PostgreSQL](https://www.postgresql.org/docs/current/ddl-partitioning.html)
+    /// [PostgreSQL Partitioning](https://www.postgresql.org/docs/current/ddl-partitioning.html)
+    /// [PostgreSQL Inheritance](https://www.postgresql.org/docs/current/ddl-inherit.html)
     fn parse_optional_create_table_config(
         &mut self,
     ) -> Result<CreateTableConfiguration, ParserError> {
+        let inherits = if dialect_of!(self is BigQueryDialect | PostgreSqlDialect | GenericDialect)
```

Comment:
the `dialect_of` macro we're transitioning away from for new code, but I think in this case there's probably no need to gate this on dialect (also not sure BQ supports this syntax so it might be misleading to explicitly name BQ here) so that we can remove the check entirely and always parse the `INHERIT` option if it shows up in the statement?

Reply:
The dialect thing must have been a copy paste from the line below and then my brain decided to drop that pointer of information - fixed as described!